src/TreeView/widget/Filters/ExclusiveFilter.js

```diff
@@ -0,0 +1,67 @@
+define([
+    "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dijit/form/DropDownButton",
+    "dijit/Menu",
+    "dijit/CheckedMenuItem",
+    "TreeView/widget/Commons",
+    "TreeView/widget/Filters/AbstractFilter"
+], function(declare, lang, DropDownButton, Menu, CheckedMenuItem, Commons, AbstractFilter) {
+    "use strict";
+
+    return declare("TreeView.widget.Filters.ExclusiveFilter", AbstractFilter, {
+        
+        itemClick: function (label, e) {
+            this.dropdown.set("label", label);
+
+            for (var i = 0; i < this.menuItems.length; i++) {
+                var item = this.menuItems[i];
+                
+                if (item.label == label) {
```

Comment:
A bit verbose? 
`item.set("checked", (item.label == label))` is more concise and would still be readable IMHO.

Reply:
Done.